packages/block-library/src/details/variations.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { chevronDown } from '@wordpress/icons';
+
+const variations = [
+	{
+		name: 'details',
+		title: __( 'Details' ),
+		description: __(
+			'An advanced block that allows displaying a summary and hiding and displaying detailed content.'
+		),
+		isDefault: true,
+		scope: [ 'inserter' ],
+		icon: chevronDown,
+	},
+	{
+		name: 'transcript',
+		title: __( 'Transcript' ),
```

Comment:
Is there any need for this variation? Could we add "transcript" to the keywords for surfaceability?

Reply:
I would refer to @joedolson here.
The transcript variation doesn't actually "do" anything right now, it has no other features other than being listed with a different name and description.

Reply:
It could be a variation; if so, I'd think it would restrict which blocks were permitted (e.g., no video/audio inside a transcript, as @poena mentioned). But surfacing it as a keyword for searchability would be fine, too, I think.